src/goto-symex/goto_symex_state.cpp

```diff
@@ -288,8 +289,7 @@ static void assert_l1_renaming(const exprt &expr)
   #if 1
   if(check_renaming_l1(expr))
```

Comment:
Is this side effect free?  If so, could we have INVARIANT(check_remaining_l1(expr), ...) ?  Maybe then we could also remove the #if 1?

Reply:
Yes, it is side-effect free (but quite costly, in actual fact).